gridmenuview/src/main/java/com/bcc/gridmenuview/task/LocalImageTask.java

```diff
@@ -10,15 +9,17 @@
 
 import com.bcc.gridmenuview.provider.ImageProvider;
 
+import java.lang.ref.WeakReference;
+
 public class LocalImageTask extends AsyncTask<String, Void, Drawable> {
 
     private Drawable drawable;
     private ImageProvider imageProvider;
-    @SuppressLint("StaticFieldLeak")
-    private Context context;
+    private final WeakReference<Context> contextWeakReference;
```

Comment:
IMO, naming the variable as `context` is just fine

Reply:
fixed mas @gifff